packages/story-editor/src/components/panels/design/layer/layer.js

```diff
@@ -112,35 +112,35 @@ const LayerButton = styled(Button).attrs({
   ${({ isSelected, theme }) =>
     isSelected &&
     css`
-      background: ${theme.colors.interactiveBg.secondaryPress};
+      background: ${theme.colors.interactiveBg.tertiaryPress};
       + * {
-        --background-color: ${theme.colors.interactiveBg.secondaryPress};
+        --background-color: ${theme.colors.interactiveBg.tertiaryPress};
         --background-color-opaque: ${rgba(
-          theme.colors.interactiveBg.secondaryPress,
+          theme.colors.interactiveBg.tertiaryPress,
           0
         )};
         --selected-hover-color: ${theme.colors.interactiveBg.tertiaryHover};
       }
     `}
 
   :hover {
-    background: ${({ theme }) => theme.colors.interactiveBg.secondaryHover};
+    background: ${({ theme }) => theme.colors.interactiveBg.tertiaryHover};
   }
   :hover + * {
     --background-color: ${({ theme }) =>
-      theme.colors.interactiveBg.secondaryHover};
+      theme.colors.interactiveBg.tertiaryHover};
     --background-color-opaque: ${({ theme }) =>
-      rgba(theme.colors.interactiveBg.secondaryHover, 0)};
+      rgba(theme.colors.interactiveBg.tertiaryHover, 0)};
   }
 
   :active {
-    background: ${({ theme }) => theme.colors.interactiveBg.secondaryPress};
+    background: ${({ theme }) => theme.colors.interactiveBg.tertiaryPress};
   }
   :active + * {
     --background-color: ${({ theme }) =>
-      theme.colors.interactiveBg.secondaryPress};
+      theme.colors.interactiveBg.tertiaryPress};
     --background-color-opaque: ${({ theme }) =>
-      rgba(theme.colors.interactiveBg.secondaryPress, 0)};
+      rgba(theme.colors.interactiveBg.tertiaryPress, 0)};
```

Comment:
apologies for the styling here being a little complicated 😅 

Looks like something is out of sync here tho with the hover styles over the layer actions:
| Staging | This Branch |
| --- | ---|
| <img width="302" alt="Screen Shot 2022-03-14 at 9 04 51 AM" src="https://user-images.githubusercontent.com/35983235/158188864-98aaa30e-2044-40ae-a8c5-67d946096ef6.png"> |  <img width="304" alt="Screen Shot 2022-03-14 at 9 05 03 AM" src="https://user-images.githubusercontent.com/35983235/158188892-7d1c5478-0f61-46ad-8886-ffe6679e2cba.png"> |

Looks like it is do to the background color of the popup changing.